google-cloud-trace/docs/toc.json

```diff
@@ -1,8 +1,71 @@
 {
-  "guides": [],
+  "guides": [
+    {
+      "title": "Troubleshooting",
+      "id": "troubleshooting",
+      "edit": "https://github.com/GoogleCloudPlatform/gcloud-common/edit/master/troubleshooting/readme.md",
+      "contents": "https://raw.githubusercontent.com/GoogleCloudPlatform/gcloud-common/master/troubleshooting/readme.md"
+    },
+    {
+      "title": "Contributing",
+      "id": "contributing",
+      "edit": "https://github.com/GoogleCloudPlatform/gcloud-common/edit/master/contributing/readme.md",
+      "contents": "https://raw.githubusercontent.com/GoogleCloudPlatform/gcloud-common/master/contributing/readme.md"
+    }
+  ],
   "services": [
     {
-      "title": "TraceServiceApi",
+      "title": "Google::Cloud",
+      "type": "google/cloud"
+    },
+    {
+      "title": "Google::Cloud::Trace",
+      "type": "google/cloud/trace",
+      "nav": [
```

Comment:
I'm guessing ruby developers would expect to see essentially a class reference in a left nav (for which the current list of 10 classes should not be unusual or difficult to use.) Even if we also have a section of the docs that is more topical, the reference section is also needed. Especially given that content comes from the yardoc, I do think it's best to structure that part of the nav accordingly.